tests/integration/nvidia_setup/test_apply_oke.py

```
@@ -0,0 +1,182 @@
+
+def test_oke_configure_limits_writes_memlock(base_image):
+    runner = DockerTestRunner(package="nvidia-setup", base_image=base_image)
+    try:
+        result = runner.run_script(
+            script="steps/configure_limits.sh",
+            configmaps={"service": "oke", "accelerator": "h100"},
+            skip_system_operations=True,
+        )
+        assert_exit_code(result, 0)
+        conf = runner.get_file_contents("/etc/security/limits.d/99-oci-hpc.conf")
+        assert "memlock" in conf and "unlimited" in conf
```

Ruff (0.15.18)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)